src/emulator/cpu/include/cpu/functions.h

```diff
@@ -17,18 +17,15 @@
 
 #pragma once
 
-#include <mem/mem.h> // Address.
+#include <cpu/state.h>
```

Comment:
Is it necessary that CPUState becomes public?

Reply:
Yeah, it's an abstract interface

Reply:
Ah sorry, i mistake your question. Yeah dont